libvast/vast/format/zeek.hpp

```diff
@@ -268,7 +268,10 @@ class writer : public format::writer {
 
   /// Constructs a Zeek writer.
   /// @param dir The path where to write the log file(s) to.
-  writer(path dir);
+  /// @param show_timestamp_tags Flag to control whether to include comments
+  /// with `#open` and `#close` tags in the output. Setting this flag to
+  /// `false` produces a deterministic output.
+  writer(path dir, bool show_timestamp_tags);
```

Comment:
Should we add `= true` as default argument, so the caller doesn't have to know details about the zeek format?

Reply:
I also thought about that and decided against that. I couldn't find a code path where it would make sense to hide this detail at the call site.